### PR TITLE
Refresh items list on filter removal

### PR DESCRIPTION
**Pull request by thecalcc**
CPCN-268
## Checklist

- [x] This pull request is not adding new forms that use redux
- [x] This pull request is adding missing TypeScript types to modified code segments where it's easy to do so with confidence

### CODE REVIEW

File: assets/search/components/SearchResultsBar/SearchResultsFiltersRow.tsx [hunk 1]
```diff
@@ -56,6 +57,7 @@ export function SearchResultsFiltersRow({readonly, searchParams, filterGroups, t
                                 ...searchParams.created,
                                 from: null,
                             });
+                            refresh?.();
```
Comment (tomaskikutis) [inline assets/search/components/SearchResultsBar/SearchResultsFiltersRow.tsx:60]: can't`setCreatedFilter` handle the refreshing?

Comment (thecalcc) [reply assets/search/components/SearchResultsBar/SearchResultsFiltersRow.tsx:60]: `setCreatedFilter` currently doesn't. Even if I made it do the refreshing it's not used in all the places so it's not suitable.

**Comment by thecalcc** [comment]
@tomaskikutis  I've changed it so that `setCreatedFilter` and `toggleFilter` dispatch the refresh inside the actions themselves. For `resetFilter` where it's being passed to `SearchResultTagsList` it's already doing the refreshing there.

File: assets/search/components/SearchResultsBar/index.tsx [hunk 2]
```diff
@@ -212,9 +212,9 @@ class SearchResultsBarComponent extends React.Component<any, any> {
                             setAdvancedSearchKeywords={this.props.setAdvancedSearchKeywords}
                             toggleFilter={this.props.toggleFilter}
                             setCreatedFilter={this.props.setCreatedFilter}
+                            resetFilter={this.resetFilter}
```
Comment (tomaskikutis) [inline assets/search/components/SearchResultsBar/index.tsx:215]: looks like this file shouldn't have changes

**Review by tomaskikutis — APPROVED** [review]
Good stuff. I said you're trying to trick me with a bad implementation ;D

**Comment by thecalcc** [comment]
> Good stuff. I said you're trying to trick me with a bad implementation ;D

you were right in the end ;D as per usual

**Pull request merged**